dhis2_android_maps/src/main/java/org/dhis2/maps/utils/GetMapData.kt

```diff
@@ -34,7 +34,7 @@ object GetMapData {
 
         val locationFeatures = locationItems
             .filter { it !is LocationItemModel.StoredResult }
-            .takeIf { captureMode.isSearch() }
+            .takeIf { captureMode.isSearch() || captureMode.isSearchSwipe() || captureMode.isSearchManual() }
```

Comment:
Maybe create a method in the Enum class that checks if captureMode is in search mode, since I see you do this check on other classes too.

Reply:
good idea!